docker/local-build.sh

```diff
@@ -39,19 +42,19 @@ echo "$BUILD_NUMBER" > $WORKSPACE/.build_number
 CI_BUILD_ID="${BUILD_TIMESTAMP}-build-${BUILD_NUMBER}"
 
 # export other vars
-for var in WORKSPACE BASE_DISTRO CURRENT_PROJECT BUILD_CACHE_DIR GIT_PROXY_COMMAND CI_BUILD_ID TARGET_MACHINE; do
-	RUN_ARGS="$RUN_ARGS -e $var=${!var}"
+for var in WORKSPACE BASE_DISTRO CURRENT_PROJECT BUILD_CACHE_DIR GIT_PROXY_COMMAND CI_BUILD_ID TARGET_MACHINE GIT_COMMITTER_NAME GIT_COMMITTER_EMAIL; do
+	RUN_ARGS+=(-e "$var=${!var}")
 done
 # Point HOME to WORKSPACE, don't polute real home.
-RUN_ARGS="$RUN_ARGS -e HOME=$WORKSPACE"
+RUN_ARGS+=(-e "HOME=$WORKSPACE")
 
 docker build -t $CURRENT_PROJECT $BUILD_ARGS $WORKSPACE/docker/$BUILDOS
 
 if [ ! -d $BUILD_CACHE_DIR ]; then
     mkdir -p $BUILD_CACHE_DIR
 fi
 
-docker run -it --rm $RUN_ARGS \
+docker run -it --rm "${RUN_ARGS[@]}" \
```

Comment:
any space in this subset of variables usually means syntax error, which would be visible if not in this script, but then further down the line. Better sanitize values near line 25.